### PR TITLE
Feat/bottom navigation

### PR DESCRIPTION
**Pull request by ioanajianu**
This merge request addresses issue #12 .
This change allows users to navigate between 3 main screens using a bottom navigation bar: 
1. Overview: Will show the list of active trips
2. Search: Will be able to search by Trip name/location/activity (TBD)
3. Profile: Will show profile customization
image

I added some default screens for each screen, a bottom navigation bar and tests.

Also, I refactored the project a bit to keep the gradle and libs versions files similar to what we used during the bootcamp.

### CODE REVIEW

File: app/src/main/java/com/android/voyageur/MainActivity.kt [hunk 1]
```diff
@@ -24,20 +32,36 @@ class MainActivity : ComponentActivity() {
...
       }
     }
   }
 }
 
 @Composable
-fun Greeting(name: String, modifier: Modifier = Modifier) {
-  Text(text = "Hello $name!", modifier = modifier.semantics { testTag = C.Tag.greeting })
-}
+fun VoyageurApp() {
```
Comment (g0g05arui) [inline app/src/main/java/com/android/voyageur/MainActivity.kt:43]: Move this to a separate file

**Review by g0g05arui — APPROVED** [review]
LGTM

File: app/src/main/java/com/android/voyageur/model/trip/Trip.kt [hunk 2]
```diff
@@ -15,19 +15,19 @@ data class Trip(
     val activities: Array<Any>, // TODO : replace this with activity model
     val type: TripType,
 ) {
-    override fun equals(other: Any?): Boolean {
-        if (this === other) return true
-        if (javaClass != other?.javaClass) return false
+  override fun equals(other: Any?): Boolean {
```
[the remark this answers is not in the document]
Comment (ioanajianu) [reply app/src/main/java/com/android/voyageur/model/trip/Trip.kt:18]: Sounds good, thanks! Wdyt about using ktfmtFormat like in the bootcamp?

File: app/src/main/java/com/android/voyageur/MainActivity.kt (hunk 1 above)
Comment (ioanajianu) [reply app/src/main/java/com/android/voyageur/MainActivity.kt]: Done

**Pull request merged**